src/Operators/spacings_and_areas_and_volumes.jl

```diff
@@ -35,6 +35,17 @@ The operators in this file fall into three categories:
 #####
 #####
 
+const ZRG = Union{LLGZ, RGZ, OSSGZ}
+
+@inline getspacing(k, Δz::AbstractVector) = @inbounds Δz[k]
+@inline getspacing(k, Δz::Number)         = @inbounds Δz
+
+@inline Δrᵃᵃᶜ(i, j, k, grid::AbstractGrid) = getspacing(k, grid.z.Δᵃᵃᶜ)
+@inline Δrᵃᵃᶠ(i, j, k, grid::AbstractGrid) = getspacing(k, grid.z.Δᵃᵃᶠ)
```

Comment:
We don't strictly need the `r` stuff in this PR right? It's ok to keep it, just want to understand.

Reply:
right, we don't need it here because at the moment `z = r`